dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/StoredCompany.kt

```diff
@@ -17,5 +17,5 @@ data class StoredCompany(
     val companyInformation: CompanyInformation,
 
     @field:JsonProperty(required = true)
-    val dataRegisteredByDataland: MutableList<DataMetaInformation>
+    val dataRegisteredByDataland: List<DataMetaInformation>
```

Comment:
Ist die Anpassung weil wir aktuell keine Daten löschen?

Reply:
Es ist eher dafür da, Konsistenz zu schaffen, weil alle anderen Attribute in den API Modellen auch Immutable sind (und wir das auch nicht benutzen)